CMakeLists.txt

```diff
@@ -55,6 +55,10 @@ function(find_pmemcheck)
 	endif()
 endfunction()
 
+if(NOT CMAKE_BUILD_TYPE)
+	set(CMAKE_BUILD_TYPE "Release")
+endif()
```

Comment:
I think we should adopt strategy similar to this: https://blog.kitware.com/cmake-and-the-default-build-type/